include/s3select.h

```diff
@@ -536,6 +536,23 @@ struct push_debug_1 : public base_action
 };
 static push_debug_1 g_push_debug_1;
 
+struct push_null : public base_action 
+{
+  void operator()(const char* a, const char* b) const
+  {
+    std::string token(a, b);
+    variable* v;
+
+    if (token.compare("null") == 0 || token.compare("NULL")==0) 
```

Comment:
push_null is called upon syntax-parser is identifying "null" or "NULL"
do, why to compare again?